src/details/ArborX_Traits.hpp

```diff
@@ -59,12 +57,11 @@ struct Access<View, typename std::enable_if<Kokkos::is_view<View>::value &&
 
   static typename View::size_type size(View const &v) { return v.extent(0); }
 
-  using Tag = PointTag;
+  using Tag = Details::PointTag;
```

Comment:
With these changes the user has to pull the geometry tags from the `Details::` namespace.
I will update #107 to reflect these changes after this PR gets merged.
My intention is to discuss our options regarding the geometry tags then.